### PR TITLE
Add CWA container diagram

### PR DESCRIPTION
**Pull request by davidkelliott**
## What does this pull request do?

Add a container diagram for CWA

## Why make these changes?

Whilst looking at migrating CWA to AWS, we need to understand the
existing architecture.  This is a cross between a container diagram and
a deployment diagram, but until a separate deployment diagram has been
done I believe having infrastructure elements on this diagram adds value.

## What is the result?

cwa-container

### CODE REVIEW

File: diagrams/get-access/cwa-containers.yaml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+- type: Software System
+  name: HUB Database
+  tags: database
+  position: '2900,1900'
+- type: Software System
+  name: LAA Portal
+  description: LAA identity management solution
+  position: '2900,1500'
+- type: Software System
+  name: V Shield Edge
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: Does it matter in this diagram what implements this? Could it be a "Firewall"? Or if it matters, "VMware vShield Edge"?

File: diagrams/get-access/cwa-containers.yaml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  description: Virtual Firewall - SSL termination happens here, this is the beginning of the LAA estate
+  tags: infrastructure
+  position: '900,300'
+- type: Software System
+  name: VXR Router
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: Same here, does it matter in this diagram that it's a VXR router? Could it be "Entrypoint Router"? (We could put the make/details into the description.)

File: diagrams/get-access/cwa-containers.yaml [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  color: '#ffffff'
+- type: element
+  tag: Person
+  shape: Person
+- type: element
+  tag: database
+  shape: Cylinder
+- type: element
+  tag: infrastructure
+  background: '#28a197'
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: In the readme, the colour scheme defines this colour as:

https://github.com/ministryofjustice/laa-architecture-documentation/blob/30c04c22d5e321fb5f76d1ceb034dc565a552ca6/README.md#L39

Could we use another one, please?

File: diagrams/get-access/cwa-containers.yaml [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  color: '#ffffff'
+- type: relationship
+  tag: AWS VPN
+  dashed: 'false'
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: 🙋‍♂ It's a bit hard to spot that from a distance that these lines are different. What do you think about giving them a different colour? Same as the infrastructure maybe?

File: diagrams/get-access/cwa-containers.yaml [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+- type: Person
+  name: External Users
+  description: Users not using DOM1 computers
+  position: '2925,450'
+- type: Person
+  name: Internal Users
+  description: Users using DOM1 computers
+  position: '2925,50'
+- type: Software System
+  name: Client Costs and Management System (CCMS)
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: _naming_
The system context diagram defines this node as `Client and Cost Management System`.

File: diagrams/get-access/cwa-containers.yaml [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+description: Container Diagram for CWA
+
+elements:
+- type: Person
+  name: External Users
+  description: Users not using DOM1 computers
+  position: '2925,450'
+- type: Person
+  name: Internal Users
+  description: Users using DOM1 computers
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: Is it important that they use DOM1 on this diagram? What do you think about calling them "Ministry of Justice staff"?

File: diagrams/get-access/cwa-containers.yaml [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+  Structurizr Express: https://structurizr.com/express
+---
+type: Container
+scope: Contracted Work and Administration (CWA)
+description: Container Diagram for CWA
+
+elements:
+- type: Person
+  name: External Users
+  description: Users not using DOM1 computers
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: What do you think about defining "external" as "they are not employed or contracted by the Ministry of Justice"?
Could you add the "external" colour scheme here too, please?

File: diagrams/get-access/cwa-containers.yaml [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+  - type: Container
+    name: Oracle HTTP Server (OHS) and Webgate
+    description: WEB TIER - Application Webgate (2 instances), intercepts traffic and redirects to the AccessGate if the user is not authenticated
+    position: '900,1100'
+- type: Software System
+  name: EDW005 Database
+  tags: database
+  position: '100,1900'
+- type: Software System
+  name: HA Proxy
```
Comment (sldblog) [inline diagrams/get-access/cwa-containers.yaml]: _naming_
What do you think about calling this by its function, like "Load Balancer"?

File: diagrams/get-access/cwa-containers.yaml (hunk 7 above)
Comment (davidkelliott) [reply diagrams/get-access/cwa-containers.yaml]: Contracted by the ministry of justice is a bit misleading as external providers have contracts with us, how about "Legal Provider" to match the system context diagram?

**Pull request merged**